lib/pal/posix/sysinfo_sources.cpp

```diff
@@ -225,6 +225,14 @@ sysinfo_sources_impl::sysinfo_sources_impl() : sysinfo_sources()
     // add("proc_loadavg", {"/proc/loadavg", "(.*)[\n]*"});
     // add("proc_uptime", {"/proc/uptime", "(.*)[\n]*"});
 
+    // osName may contain quotes on openSUSE
+    if (get("osName").find('"') == 0)
```

Comment:
It looks Okay. You could have also probably just patched the contents of cache, because `add` would force a file re-read. But it's a minor thing anyways.

Reply:
Fixed.